nixos/modules/services/desktop-managers/cosmic.nix

```diff
@@ -162,6 +162,7 @@ in
     services.avahi.enable = lib.mkDefault true;
     services.gnome.gnome-keyring.enable = lib.mkDefault true;
     services.gvfs.enable = lib.mkDefault true;
+    services.orca.enable = lib.mkDefault true;
```

Comment:
Most other DEs that enable it do something like the following, so we probably should too?

```nix
services.orca.enable = mkDefault (notExcluded pkgs.orca);
```